### DISCUSSION TITLE
Ubuntu18安装fealpy失败

### DISCUSSION THREAD

**Discussion by EmilyDuan** [weihuayi/fealpy#50] [Installation]
按照项目主页Ubuntu相应的install方法进行安装，到第三步“$ sudo -H pip3 install ./”时出错，错误提示为“error: command 'x86_64-linux-gnu-gcc' failed with exit status 1”“Failed building wheel for MeshPy
”。在网上查找许久，尝试多种方法，无果。但又很想使用此包，来进行有限元的学习。
请问老师，有遇到过类似情况，有解决办法么？
谢谢！期待进一步指导～

**Comment by weihuayi** [comment]
你是 Ubutnu 18.04 吗?

**Comment by EmilyDuan** [comment]
是的，不过刚刚解决了这个问题，原因是缺少pybind11，pip3 install pybind11之后就没这个问题了～

现在出现了新的问题，在fealpy目录下执行sudo -H pip3 install ./，运行成功。虽然/usr/local/lib/python3.6/dist-packages有了fealpy，但是是空的，并不能import fealpy.mesh之类的。再重新运行一次，提醒 requirement already satisfied.但实际上dist-packages目录下并没有完整的fealpy文件夹。

多谢老师的回复，我再找找现在问题的原因:)～
